src/EdFi.AnalyticsMiddleTier.DataStandard31/Asmt/MSSQL/0014-View-StudentAssessmentFact-Create.sql

```diff
@@ -0,0 +1,118 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'asmt_StudentAssessmentFact')
+BEGIN
+	DROP VIEW analytics.asmt_StudentAssessmentFact
+END
+GO
+
+CREATE VIEW analytics.asmt_StudentAssessmentFact AS
+
+	SELECT
+        CONCAT(StudentAssessment.AssessmentIdentifier, '-', StudentAssessment.Namespace, '-', StudentAssessment.StudentAssessmentIdentifier, '-', StudentAssessment.StudentUSI) AS StudentAssessmentKey,
+		CASE WHEN SASOA.StudentUSI IS NULL THEN NULL ELSE CONCAT(SASOA.StudentUSI, '-', SASOA.IdentificationCode, '-', SASOA.AssessmentIdentifier, '-', SASOA.StudentAssessmentIdentifier, '-', SASOA.Namespace) END AS StudentObjectiveAssessmentKey,
```

Comment:
Can you format the case statement a little bit better to make it more readable? There are other views where we have case statements as well, maybe you can find them and use them as a reference.

Reply:
resolved, please review it again.